### PR TITLE
Improve translation

### PR DESCRIPTION
**Pull request by ZeHiro**
[Review on Reviewable](https://reviewable.io/reviews/bnjbvr/kresus/335)

### CODE REVIEW

File: shared/locales/fr.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ module.exports = {
 
         charts: {
             Amount: 'Montant',
-            balance: 'balance',
-            by_category: 'par catégorie',
-            differences_all: 'rentrées et sorties (tous les comptes)',
+            balance: 'Solde',
+            by_category: 'Par catégorie',
+            differences_all: 'Rentrées et sorties (tous les comptes)',
```
Comment (bnjbvr) [inline shared/locales/fr.js]: I like the fact that it isn't capitalized, but that might be just me. However, if you change capitalization, can you also change it in the `en` locale?

Comment (ZeHiro) [inline shared/locales/fr.js]: I prefer it capitalized, but it's a matter of tastes :p I can keep it without capitalization. Ate you OK with the 'relevé' translation (singulier)

**Comment by bnjbvr** [comment]
Thanks for the update! Can you rebase, please?

**Comment by bnjbvr** [comment]
Thank you!

**Pull request merged**